starlite_jwt/jwt_auth.py

```diff
@@ -4,7 +4,8 @@
 from pydantic import BaseConfig, BaseModel
 from pydantic_openapi_schema.v3_1_0 import SecurityScheme
 from starlette.status import HTTP_201_CREATED
-from starlite import DefineMiddleware, MediaType, Response
+from starlite import DefineMiddleware, Response
+from starlite.enums import MediaType
```

Comment:
?

Reply:
so this was the example I mentioned in discord - it make the cross-reference of the type work because we have it referenced in starlite api docs as `starlite.enums.MediaType`.. I just included to demonstrate.